src/main/kotlin/com/semgrep/idea/settings/LspSettings.kt

```diff
@@ -23,6 +23,33 @@ data class SemgrepLspSettings(
     var heapSizeJS: Int = 4096,
 ) {
 
+    fun toFlatMap(): Map<String, String> {
```

Comment:
`flatMap` has meaning in the JVM ecosystem -- `toFlattenedMap` might be more appropriate